gemma-core/src/main/java/ubic/gemma/persistence/service/expression/experiment/FactorValueService.java

```diff
@@ -52,12 +53,18 @@ public interface FactorValueService extends BaseService<FactorValue>, FilteringV
     @Secured({ "GROUP_USER", "ACL_SECURABLE_EDIT" })
     void remove( FactorValue factorValue );
 
+    /**
+     * This method accepts a {@link Characteristic} for compatibility with existing code.
```

Comment:
It would be worth to mention that passing a non-statement to this method will raise an exception.

Reply:
Ok, it now accept either and will remove onejcts of applicable.